### ISSUE TITLE
training pytorch model on celebAHQ

### ISSUE THREAD

**Issue by ljjcoder** [shepnerd/inpainting_gmcnn#20]
When I training the pytorch model on celebAHQ from the pretrain model you give, I watch the tensorboard result  seems to be blurred like this.
![1561199895(1)](https://user-images.githubusercontent.com/40750189/59962851-7c174200-951d-11e9-9779-c32147f90ec4.png)
I train the model on 1080ti with batch size 4. could you give me some sugestion to get more real results?

**Comment by shepnerd** [comment]
I guess you may train the model with only the reconstruction loss. To train the model with all losses, you need set option `pretrain_network` as 0. Specifically, just as the follows:
`python train.py ... --pretrain_network 0 ...`.

**Comment by ljjcoder** [comment]
@shepnerd thanks for your reply, when I change set pretrain_network 0, it will produce perfect results training from pretrain model on rect mode. but when I train the stoke mode from scrach ,the reuslt seems to  be very bad like this. Could you give me some sugestion to get more real results and could you give the stoke pretrain model to me? thanks.



![1561361548(1)](https://user-images.githubusercontent.com/40750189/60000108-1c907200-9696-11e9-85a4-c77561120765.png)

**Comment by shepnerd** [comment]
1. pretrain the model first (--pretrain_network 1)
2. finetune the model with all losses when the previous training stage converges (--pretrain_network 0 --load_model_dir ...)

We do not have the pretrained model on celeba-hq using strokes with pytorch implementation. We may add it in the future. Maybe you could try our tensorflow version.

**Comment by ljjcoder** [comment]
@shepnerd I find the places2 model you give is trained on stroke, could you give me the model trained on rect?

**Comment by shepnerd** [comment]
We may release places2 model trained with rectangle masks in the future.
Currently, you could just finetine model trained with strokes using
rectangle masks.

ljjcoder <notifications@github.com> 于2019年6月27日周四 下午5:02写道：

**Comment by cmyyy** [comment]
Hello, @ljjcoder, i met some problem when using tensorboard to do visualization.
Could you share the envs of visualization, like the versions of tensorboard, tensorboardX, tensorflow etc.. Thanks a lot.
